<launch>
    <node pkg="platforms_server" name="vrep_communicator_node" type="vrep_communicator_node.py" output="screen"/>
    <node pkg="platforms_server" name="path_planner_node" type="path_planner_node.py" output="screen"/>
</launch>
